### PR TITLE
Fine-grained control of some permissions

### PR DESCRIPTION
**Pull request by yang1666204**
## Summary

1. Fine-grained control of some permissions.
2. Iops limit.
3. Optimize the UI of Resource pool Modal.
image
4. Improve the verification of role permissions.
image

### CODE REVIEW

File: ui/src/components/SelectNSFromItem/index.tsx [hunk 1]
```diff
@@ -25,28 +29,36 @@ export default function SelectNSFromItem({
...
-        >
-          <PlusOutlined />
-          <span style={{ marginLeft: '6px' }}>
-            {intl.formatMessage({
-              id: 'OBDashboard.Cluster.New.BasicInfo.AddNamespace',
-              defaultMessage: '新增命名空间',
-            })}
-          </span>
-        </div>
+        {hasSysAccess ? (
```
Comment (powerfooI) [inline ui/src/components/SelectNSFromItem/index.tsx:32]: `system:write` is required to create a namespace. The above `access.systemread || access systemwrite` is incorrect.

File: ui/src/components/customModal/ModifyUnitDetailModal.tsx [hunk 2]
```diff
@@ -501,30 +495,30 @@ export default function ModifyUnitDetailModal({
...
+                  defaultMessage: '请输入',
+                })}
+              />
+            </Form.Item>
+          </Col>
+          <Col span={8}>
+            <Form.Item label="max iops" name={['unitConfig', 'maxIops']}>
+              <InputNumber
+                style={{ width: 222 }}
+                min={minResource.maxIops}
```
Comment (powerfooI) [inline ui/src/components/customModal/ModifyUnitDetailModal.tsx:514]: Does this mean that `maxIops` can not be decreased?

File: ui/src/pages/Alert/Shield/index.tsx [hunk 3]
```diff
@@ -289,7 +293,7 @@ export default function Shield() {
           </h2>
         }
         extra={
-          access.alarmwrite ? (
+          access.alarmwrite || !access.obclusterread ? (
```
Comment (powerfooI) [inline ui/src/pages/Alert/Shield/index.tsx:296]: Why can a user create a shield without `obcluster:read` permission? I thought `access.alarmwrite && access.obclusterread` should be correct.

File: ui/src/pages/OBProxy/index.tsx [hunk 4]
```diff
@@ -53,17 +54,21 @@ export default function OBProxy() {
...
+        ) : null}
       </Row>
-      <MonitorComp
-        filterLabel={[]}
-        queryScope="OBPROXY"
-        type="OVERVIEW"
-        groupLabels={['cluster']}
-        queryRange={DEFAULT_QUERY_RANGE}
-      />
+      {access.systemread || access.systemwrite ? (
```
Comment (powerfooI) [inline ui/src/pages/OBProxy/index.tsx:63]: Why add `system` permission check here while remove the check in `pages/Cluster/index.tsx`?

File: ui/src/components/customModal/ModifyUnitDetailModal.tsx (hunk 2 above)
Comment (yang1666204) [reply ui/src/components/customModal/ModifyUnitDetailModal.tsx:514]: I remember there is a minimum value for resources (minResource). If it is less than this value, the resource pool cannot be created. So the input box here limits the minimum value.

Comment (powerfooI) [reply ui/src/components/customModal/ModifyUnitDetailModal.tsx:514]: OK

File: ui/src/pages/OBProxy/index.tsx (hunk 4 above)
Comment (yang1666204) [reply ui/src/pages/OBProxy/index.tsx:63]: This is missing, the check for system should be removed.

**Review by powerfooI — APPROVED** [review]
LGTM

**Pull request merged**